<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manage_paper_part</class>
 <widget class="QWidget" name="Manage_paper_part">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="changeBtn">
     <property name="text">
      <string>修改试卷</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deleteBtn">
     <property name="text">
      <string>删除试卷</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="copyBtn">
     <property name="text">
      <string>复制试卷</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
